code401challenges/array-reverse/README.md

```diff
@@ -0,0 +1,16 @@
+# Reverse an Array
+<!-- Short summary or background information -->
+Given the integer array, reverse the elements of the array. 
+## Challenge
+<!-- Description of the challenge -->
+Challenge was to how to determine the hafway mark using for loop when the array length is odd number.
+
+## Approach & Efficiency
+<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
+Instead of creating a new array which would have been easier to do, we reversed the same array. Created temp variable and chained the elements to reverse the array.
+
+## Solution
+<!-- Embedded whiteboard image -->
+![Image](Image from iOS.jpg)
```

Comment:
Note that this embedded image isn't working :( You'll need to use something more like 
```
![Image](./assets/Image from iOS.jpg)
```
(I also wonder if the issue is having spaces in your filename... but try that version of the path to the image first.)